source/channels/rename-channels.rst

```diff
@@ -5,10 +5,27 @@ Rename channels
   :start-after: :nosearch:
 
 Anyone can rename the channels they belong to, unless the System Admin has `restricted the permissions to do so using advanced permissions </onboard/advanced-permissions.html>`__.
+.. tabs::
 
-In the desktop app, select the channel name at the top of the center pane to access the drop-down menu, then select **Rename Channel**. You'll be prompted for two values:
+  .. tab:: Web browser/Desktop app
 
-- **Channel display name:** This appears in the Mattermost user interface. 
-- **Channel handle:** This is part of the channel URL. You can also change the channel handle when renaming a channel, but changing channel handles may break existing links.
+      **To Rename  channel in the desktop app**
+
+        In the desktop app, select the channel name at the top of the center pane to access the drop-down menu, then select **Rename Channel**. You'll be prompted for two values:
+
+        - **Channel display name:** This appears in the Mattermost user interface. 
+        - **Channel handle:** This is part of the channel URL. You can also change the channel handle when renaming a channel, but changing channel handles may break existing links.
+
+        For example, for the following channel, ``https://community.mattermost.com/core/channels/ux-design``, The channel display name would be ``UX Design``, and the channel handle would be ``ux-design``.
+
+    .. tab:: Mobile app
+       
+        **To Rename  channel in the mobile app V2**
+
+           In the mobile app, select the channel name from the sidebar to access the menu, then select **Edit Channel** option. There will be three values:
+           - **Name:** you can give a suitable name which will appear in the Mattermost user interface.
+           - **Purpose:** you can write the motive of the channel.
+           - **Header:** we can use it to list various links and repos.
```

Comment:
```suggestion
           - **Header:** This may include information relevant to the channel, such as useful contacts or documents. This is optional.
```